internal/terraform/tfswitcher/install.go

```diff
@@ -15,7 +15,7 @@ import (
 const (
 	installFile               = "terraform"
 	versionPrefix             = "terraform_"
-	installPath               = ".terraform.versions"
+	installPath               = ".ize/.terraform.versions/"
 	recentFile                = "RECENT"
 	defaultBin                = "/usr/local/bin/terraform" //default bin installation dir
```

Comment:
See my comment about tfpath and installation above. I don't think we need to use that feature, but instead run the required version directly from the downloaded path or something along those lines.